### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by markedwardmurray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,291 @@
...
+| `SK`   | Scene Kit       | Rendering 3D objects and effects. |
+| `CL`   | Core Location   | Handling location data on a GPS-capable device. |
+| `MK`   | Map Kit         | Integrating with Apple Maps. |
+
+Each of these libraries contains hundreds or even thousands of prefabricated class files. Because many of them are very large and have rather specific purposes, any library other than Core Foundation is not typically "imported" (or loaded) into a class file until the additional functionality of that supplemental library is required.
+
+### Custom Class Prefixing
+
+You may have noticed the `FIS` prefix on the `FISAppDelegate` class in previous labs; these are the initials for "**F**lat**i**ron **S**chool", and is the customized prefix that we use in all of our Objective-C labs. It's best practice to prefix custom classes in order to disambiguate them from the classes in other libraries. While it's unlikely that you'd create a custom class that coincides with a class in an Apple framework, name collisions become a very real possibility when importing third party frameworks found in the Open Source community (i.e. via CocoaPods like the Specta and Expecta testing frameworks).
```
Comment (jmburges) [inline README.md:39]: Mention that user generated content should have a three letter prefix. two letter is for apple things

File: README.md [hunk 2]
```diff
@@ -1,3 +1,291 @@
...
+
+Imagine that you're building a messaging app. Without class prefixing, you might create your own custom `Message` class. Now, if you decide to import another library that didn't use a class prefix either and which also includes a `Message` class, your application won't be able to distinguish between the two causing a slew of problems. However, if both you and the third party developer whose library you imported uniquely prefixed both `Message` classes (e.g. `FISMessage` and `JBMessage`), then the filenames would be unique. Furthermore, you might find yourself working with two libraries which both include a `Message` class, so it's similarly important that the classes are distinct:
```
Comment (jmburges) [inline README.md:41]: just making sure this reading happens after "what is a class"

Comment (markedwardmurray) [inline README.md:41]: Yes, this is intended to be the fourth reading, following Intro To OOP, Class vs Instance, and Property Basics.

File: README.md (hunk 1 above)
Comment (markedwardmurray) [inline README.md:39]: I did in the note on line 48. Do you think that should come earlier?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,291 @@
...
+
+[JBMessage.h](https://github.com/josipbernat/JBMessage/blob/master/JBMessage/JBMessage/JBMessage.h)  
+[TSMessage.h](https://github.com/KrauseFx/TSMessages/blob/master/Pod/Classes/TSMessage.h)
```
Comment (jmburges) [inline README.md:44]: AWESOME concrete example

File: README.md (hunk 1 above)
Comment (jmburges) [inline README.md:39]: oops. ignore. I had to keep reading :)

File: README.md [hunk 4]
```diff
@@ -1,3 +1,291 @@
...
+
+**Note:** *Apple reserves the authority to use any two-letter prefix for new libraries at future dates. For this reason, they suggest that third-party developers choose three-letter prefixes because there is no chance that Apple will create a future library with that exact prefix. This guideline is not always followed in the Open Source community, but it is considered best practice to use three-letter prefixes.*
+
+## A Time To Subclass
+
+Despite the plethora of class files available to us, solving any specific problem typically requires creating at least a few custom classes to tailor-fit our code to the current situation. None of the Apple libraries include, for example, a `Warship` class, nor a `ToySoldier` class, nor a `ChessBoard` class. In order to build an application that requires a representation of one of these objects, we would have to create a custom class to hold the data (properties) and perform the behaviors (methods) that the representation requires.
```
Comment (jmburges) [inline README.md:52]: In the end, almost all apps live within some sort of domain that is custom to them. A domain is the general area that your app lives in. Is it a shopping cart app? That means you'll need to model things like a `cart` and an `item`. no programming language could ever come with all classes for all possible problems! It is your job as a developer to model that domain, to bring all of the custom bits of the domain into your project

File: README.md [hunk 5]
```diff
@@ -1,3 +1,291 @@
...
+
+Creating a custom class is called "subclassing" and as a practice relies heavily on the concept of inheritance. Since we're not going to discuss the concept of inheritance until later, right now we're only going to subclass from `NSObject`—the most basic object of all of the objects; even it's name is boring.
+
+**//Flat-fact:** *The word "ponso" is derived from the acronym PONSO meaning "Plain Old NSObject".*
```
Comment (jmburges) [inline README.md]: "You may see the phrase "ponso" or "PONSO" online...

File: README.md [hunk 6]
```diff
@@ -1,3 +1,291 @@
...
+// FISWarship.m
+
+#import "FISWarship.h"
+
+@implementation FISWarship
+
+@end
+```
+
+You may notice that library was imported using angle brackets (`<` `>`) while the individual header file was imported using double quotes (`"` `"`). This is a standard syntax for distinction: each library has its own library header file that simply contains all of the header files for every class in the library.
```
Comment (jmburges) [inline README.md]: "code you write comes in with `"`, code someone else wrote comes in with `<`

**Comment by jmburges** [comment]
one missing link, and :shipit:

**Pull request merged**